### ISSUE TITLE
Edit Added Users

### ISSUE THREAD

**Issue by ArchBlood** [humhub/mail#244]
See the following post on the community: https://community.humhub.com/content/perma?id=249368

**Comment by luke-** [comment]
The Creator of the Message should have this option.

**Comment by Kilian191** [comment]
I can add users to a conversation, but is there a way to remove them? We have workgroups where users are added, but if they leave the group because they have other tasks, I can't manually remove them from the group—and not everyone uses the "Leave conversation" option.

**Comment by ArchBlood** [comment]
> I can add users to a conversation, but is there a way to remove them? We have workgroups where users are added, but if they leave the group because they have other tasks, I can't manually remove them from the group—and not everyone uses the "Leave conversation" option.

That is correct, currently there is no way implemented for managed participants of a conversation, only the leave option is implemented, I believe the reason why it hasn't been implemented is because there was no use case currently.
